chain/ethereum/src/env.rs

```diff
@@ -157,6 +162,8 @@ struct Inner {
     max_block_range_size: BlockNumber,
     #[envconfig(from = "GRAPH_ETHEREUM_JSON_RPC_TIMEOUT", default = "180")]
     json_rpc_timeout_in_secs: u64,
+    #[envconfig(from = "GRAPH_ETHEREUM_BLOCK_RECEIPTS_TIMEOUT", default = "180")]
```

Comment:
I think this waaaaayyyy too long for startup, I would probably go with something like 5 or 10 seconds